lib/l10n/intl_en_US.xml

```diff
@@ -3098,27 +3098,55 @@
     description="Text label for Drafts destination."
     >Drafts</string>
   <string
-    name="demoTwoPaneDualScreenLabel"
-    description="Option title for TwoPane demo in dual screen mode."
-    >Dual Screen</string>
+    name="demoTwoPaneFoldableLabel"
+    description="Option title for TwoPane demo in foldable mode."
+    >Foldable</string>
   <string
-    name="demoTwoPaneDualScreenDescription"
-    description="Description for the dual screen option configuration on the TwoPane demo"
-    >This is how it behaves on a dual screen device.</string>
+    name="demoTwoPaneFoldableDescription"
+    description="Description for the foldable option configuration on the TwoPane demo."
+    >This is how it behaves on a foldable device.</string>
   <string
     name="demoTwoPaneSingleScreenLabel"
-    description="Option title for TwoPane demo in single screen mode."
+    description="Option title for TwoPane demo in single screen mode. Counterpart of the foldable option."
     >Single Screen</string>
   <string
     name="demoTwoPaneSingleScreenDescription"
-    description="Description for the single screen option configuration on the TwoPane demo"
-    >This is how it behaves on a single screen device.</string>
+    description="Description for the single screen option configuration on the TwoPane demo."
+    >This is how TwoPane behaves on a single screen device.</string>
```

Comment:
Might make more sense to replace with "small screen" everywhere, since tablets and desktops are also single screen devices